package.json

```diff
@@ -3629,7 +3629,7 @@
 		"test": "npm run build && npm run build-tests && npm run instrument-dist && npm run instrument && npm run test-only && npm run report_lcov && npm run report_screen",
 		"instrument-dist": "cd out/dist && nyc instrument --compact false --in-place . . && cd ../..",
 		"instrument": "cd out/src && nyc instrument --compact false --in-place . . && cd ../..",
-		"test-only": "node ./out/src/test/test_all.js && npm run test-grammar",
+		"test-only": "node ./out/src/test/test_all.js",
```

Comment:
This doesn't seem to work. On Windows, I ran:

```
npm test .\src\test\dart\dartdoc
```

However `DART_CODE_TEST_FILTER` is never set and I don't see the output from:

```
console.log(`Running tests with filter(s): ${testFilterArgs.join(", ")}`);
```

This suggests the arguments are not going through `npm test` and to `npm test-only`?